rele/management/discover.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+from django.apps import apps
+from django.utils.module_loading import module_has_submodule
+
+logger = logging.getLogger(__name__)
+
+
+def discover_subs_modules():
```

Comment:
How about naming this `_subs_modules`?

Reply:
And using this function like `discover._subs_modules`?

Reply:
But accessing to an external function whose name starts with `_` isn't a good practice, is it? 😕